contracts/hts-precompile/IHederaTokenService.sol

```diff
@@ -157,6 +157,18 @@ interface IHederaTokenService {
     struct TokenInfo {
         /// The hedera token;
         HederaToken hedera;
+        
+        /// The number of tokens (fungible) or serials (non-fungible) of the token
```

Comment:
I've updated the order of the fields to match that of the tuple [here](https://github.com/hashgraph/hedera-services/blob/master/test-clients/src/main/java/com/hedera/services/bdd/suites/utils/contracts/precompile/HTSPrecompileResult.java#L451) because I was getting an error on GetTokenInfo.